plugins/GSdx/GSHwHack.cpp

```diff
@@ -629,23 +629,6 @@ bool GSC_TombRaiderUnderWorld(const GSFrameInfo& fi, int& skip)
 	return true;
 }
 
-bool GSC_FFVIIDoC(const GSFrameInfo& fi, int& skip)
```

Comment:
Might worth to add in the commit message that it was related/is handled by OI_DoubleHalfClear.